Library/Formula/lua.rb

```diff
@@ -65,6 +73,39 @@ def install
     ln_s "#{include}", "#{include}/lua5.2"
     ln_s "#{lib}/pkgconfig/lua.pc", "#{lib}/pkgconfig/lua5.2.pc"
     ln_s "#{lib}/pkgconfig/lua.pc", "#{lib}/pkgconfig/lua-5.2.pc"
+
+    # This resource must be handed after the main install, since there's a lua dep.
```

Comment:
Crumbs. Well spotted. I read that line about 10 times and still missed the typo :crying_cat_face: